appinventor/blocklyeditor/src/exportBlocksImage.js

```diff
@@ -376,12 +377,14 @@ PNG.prototype.processData_ = function(data) {
     return data.slice(chunkStart, chunkStart + length);
   }
   this.chunks = [];
+  var phy = [112,72,89,115];
   while (chunkStart < data.length) {
     var length = decode4();
     var type = read4();
     var chunkData = readData(length);
     chunkStart += length;
     var crc = decode4();
+    if (type == 'IDAT') {this.chunks.push(new PNG.Chunk(9, 'pHYs', pHY_data, crc32(phy.concat(pHY_data)))); }
```

Comment:
Code style: each statement should be on its own line.

Reply:
Got it!